channels.c

```diff
@@ -2315,7 +2315,7 @@ channel_post_mux_listener(struct ssh *ssh, Channel *c)
 		close(newsock);
 		return;
 	}
-	if ((euid != 0) && (getuid() != euid)) {
+	if ((euid != ROOT_UID) && (getuid() != euid)) {
```

Comment:
this and similar changes will be a headache since they'll constantly conflict with changes as we bring them in from openbsd.  After the 8.9 release I'll see if openbsd would be willing to turn these into functions representing finer grained permissions (can_change_uid(), can_bind_low_ports() or similar).  This would also help for, eg, POSIX capabilities.

Reply:
Code like this found its way into several open source ports, but indeed tends to break again now and then, as the 0 for root is way too deep in UNIXers genetics

Reply:
From a code-review perspective, 0 is a magic number in this case instead of a constant meaning nothing bad happened.